termios/termios_bsd.go

```diff
@@ -73,10 +73,10 @@ func Tcflush(fd, which uintptr) error {
 }
 
 // Cfgetispeed returns the input baud rate stored in the termios structure.
-func Cfgetispeed(attr *syscall.Termios) uint64 { return attr.Ispeed }
+func Cfgetispeed(attr *syscall.Termios) uint32 { return attr.Ispeed }
```

Comment:
This was because of a build error:

term/termios/termios_bsd.go:76: cannot use attr.Ispeed (type uint32) as type uint64 in return argument
term/termios/termios_bsd.go:79: cannot use attr.Ospeed (type uint32) as type uint64 in return argument

The docs on Termios made me think uint32 was right:

https://golang.org/pkg/syscall/#Termios